notes_test.go

```diff
@@ -59,15 +58,15 @@ func (s *NotesAPISuite) TestNotesServiceCreateNoteShouldReturnNote() {
 	s.NotNil(res)
 }
 
-func (s *NotesAPISuite) TestNotesServiceGetNoteShouldReturnError() {
+func (s *NotesAPISuite) GetNoteShouldReturnError() {
 	res, err := s.srv.GetNote(context.TODO(), &notespb.GetNoteRequest{})
 	s.Require().Error(err)
 	s.Equal(status.Code(err), codes.InvalidArgument)
 	s.Nil(res)
 }
 
 /*
-func (s *NotesAPISuite) TestNotesServiceGetNoteShouldReturnNoError() {
+func (s *NotesAPISuite) GetNoteShouldReturnNoError() {
 	noteId, err := uuid.NewRandom()
```

Comment:
I guess checking if we can get a real Note in the memdb would make more sense (than to just testing if the function recognize that it is a valid UUID and not a stored Note's UUID)

Reply:
Yes of course it was just a commented test not a functional one but right